frontend/src/components/plantcard/PlantCard.js

```diff
@@ -0,0 +1,43 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+import monsteraImage from '../../assets/plant-images/3.jpg'
+
+export default function PlantCard({ name, species }) {
+  PlantCard.propTypes = {
+    name: PropTypes.string.isRequired,
+    species: PropTypes.string.isRequired,
+  }
+
+  return (
+    <CardWrapper>
+      <ImgContainer src={monsteraImage} />
+      <PlantName>{name}</PlantName>
+      <PlantSpecies>{species}</PlantSpecies>
+    </CardWrapper>
+  )
+}
+
+const CardWrapper = styled.div`
+  display: grid;
+  justify-items: center;
+`
+
+const ImgContainer = styled.div`
+  background-image: url(${(props) => props.src});
+  background-position: center;
+  background-size: cover, contain;
+  border-radius: 50%;
+  height: 100px;
+  margin-bottom: 10px;
+  width: 100px;
+`
+
+const PlantName = styled.p`
```

Comment:
Semantically, I don't think `p` is quite right here. Maybe an `h` or a `span` fits better? For `PlantSpecies` it's the same.

Reply:
h5 and h6 it is... merci